docs/source/guides/write-custom-instruction.md

```diff
@@ -31,7 +31,7 @@ Feature: Bridge feature
 
 ### 2. Extract the algorithm
 
-As you can see - **Then** section contains expected output of our **When** instruction.
+As you can see, **Then** section contains expected output of our **When** instruction.
```

Comment:
This guide should be removed unconditionally. Having custom instructions goes against the principles by which we decided to add WASM, and RISC + Vector extensions architecture that we've chosen.

Reply:
ok, removing